src/components/Map/map.js

```diff
@@ -236,10 +240,12 @@ class Map extends Component {
         "text-color": "black",
         "text-halo-color": "#ffffff",
         "text-halo-width": 2
-      }
+      },
     });
 
-    this.addClusterClickHandlerToMapLayer(clusterName);
+    // this.addClusterClickHandlerToMapLayer(clusterName);
+    this.addClusterClickHandlerToMapLayer("clusterCircle");
+    this.addClusterMouseOverHandlerToMapLayer(clusterName);
```

Comment:
Consider attaching both handlers to the "clusterCircle" layer. Currently the popup only shows up when hovering over the number, which is a smaller target than the cluster icon.